articles/primeagen-takes-as-fakes-as-his-mustache/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Primeagen Takes as Fakes as his Mustache"
```

Comment:
A title like "New Study Shows that ThePrimeagen's Takes as Fake as his Mustache" sounds more like something you'd find on a news site

Not to say that you are, but the current format sounds like you're just saying something bad about someone, especially for people who don't know who ThePrimeagen is

Reply:
fair play. will adjust